docs/excel/excel-add-ins-worksheets.md

```diff
@@ -295,6 +295,56 @@ function onWorksheetChanged(eventArgs) {
 }
 ```
 
+## Sorting events (preview)
+
+> [!NOTE]
+> The APIs for these sort-related events are currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+The `onColumnSorted` and `onRowSorted` events indicate when any worksheet data is sorted. These events fire whether the sorting is done through UI controls (like table drop-downs) or programmatically. These events are connected to individual `Worksheet` objects and the workbook's `WorkbookCollection`.
+
+> [!NOTE]
+> `onColumnSorted` fires when columns are sorted as the result of a left-to-right sort operation. `onRowSorted` fires when rows are sorted as the result of a top-to-bottom sort operation. Sorting a table using the drop-down on a column header results in an `onRowSorted` event. The event corresponds with what is moving, not what is being considered as the sorting criteria.
```

Comment:
The style guide won't load for me right now, but IIRC, "drop-down" isn't quite the right terminology. Suggest you look up the recommended term in the style guide (if it'll load for you).

Reply:
I double-checked and drop-down should be okay for developers.

![image](https://user-images.githubusercontent.com/38896772/64637736-44e85c80-d3b9-11e9-850e-34ae087bf9c6.png)

Reply:
Though in re-reading that, I'll need to clarify the control type. I'll figure out how best to describe that.

Reply:
I think if you choose to use that term, then it might be "dropdown"

Reply:
I think the best way to describe that Excel control is as a "drop-down menu", like in the above screenshot.